eng/packaging.targets

```diff
@@ -41,15 +41,6 @@
 
   <ItemGroup Condition="'$(EnablePackageValidation)' == 'true'">
     <PackageReference Include="Microsoft.DotNet.PackageValidation" Version="$(MicrosoftDotNetPackageValidationVersion)" PrivateAssets="all" IsImplicitlyDefined="true" />
```

Comment:
Should we continue referencing the PackageValidation package? I don't think we have set it up to dependency flow yet so its version never changes currently.

Reply:
> Should we continue referencing the PackageValidation package?

i think we should keep doing that, so we can test the most recent changes. 

>  I don't think we have set it up to dependency flow yet so its version never changes currently.

i will go ahead and set up that as well